client/components/gravatar/index.jsx

```diff
@@ -64,10 +73,16 @@ module.exports = React.createClass( {
 		}
 
 		const alt = this.props.alt || this.props.user.display_name;
-		const avatarURL = this.getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
+
+		const avatarURL = this.props.tempImage ||
+			this.getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
```

Comment:
> Given the fact that using booleans to get a value is a well-known pattern

it's a well-known pattern but one I'm not a big fan of, especially for getting default values. it works because of a language oddity that's not common among most other languages. it also doesn't provide any indication of why we're doing it, doesn't indicate clearly if we are trying to get a boolean-ish value in response or just the-first-of-many-possible-values